macros/schema_tests/column_values_basic/expect_column_values_to_have_consistent_casing.sql

```diff
@@ -0,0 +1,36 @@
+{% test expect_column_values_to_have_consistent_casing(model, column_name, display_inconsistent_columns=False) %}
+
+with test_data as (
+
+     select
+     distinct {{ column_name }} as distinct_values
```

Comment:
Nitpick: please indent the statements under keywords, e.g.
```sql
select
    distinct {{ column_name }} as distinct_values
from
    {{ model }}
```
throughout the macro. Thanks!